### DISCUSSION TITLE
DDfM (Docker Desktop for Mac) licensing

### DISCUSSION THREAD

**Discussion by boxleytw** [rancher-sandbox/rancher-desktop#1406] [Q&A] [answered]
I'm working in a team evaluating the fiasco of DDfM (Docker Desktop for Mac) licensing changes, and suggesting alternatives.

My searching in Rancher Desktop did not lead me to a quick answer (_mea culpa_).  My tech spidey senses says Rancher Desktop is an approach to a slice of the problem, but may not address a complete replacement for DDfM.

Are there documents I can use in helping me understand how Rancher Desktop would replace DDfM?

**Comment by jandubois** [comment, marked as answer]
> My tech spidey senses says Rancher Desktop is an approach to a slice of the problem, but may not address a complete replacement for DDfM.

I think this is correct, it is not a drop-in replacement that will work in all situations.

Rancher Desktop is a much newer project, and still dealing with many of the early issues that gets sorted out over time based on user feedback. Docker Desktop has over 5 years history, and is obviously much more mature, so you will find various areas that it implements, that are still lacking in Rancher Desktop.
 
> Are there documents I can use in helping me understand how Rancher Desktop would replace DDfM?

I think you have to approach it from the other side: define all the requirements you have, and then check if Rancher Desktop covers them.

I suspect you will find that especially in areas like networking and volume management there are various scenarios that don't work yet in Rancher Desktop. You will have to try it out, if any of your use cases run into those limitations, and decide if you can work around them until they are fixed. You will not be able to determine this from documentation alone, which is similarly still lacking some depth. It is hard to create documentation when the project is still changing very rapidly.

Join the Slack channel and talk with other early adopters about how things are working for them, and if you have similar requirements.
